### PR TITLE
USHIFT-1848: add periodic test scenario for the standard tests on rpm-based system

### PR DESCRIPTION
**Pull request by dhellmann**
Depends on #2568

/hold

### CODE REVIEW

**Comment by dhellmann** [comment]
Rebased to see if that helps with the flakes. The SELinux failure looks like it might be real because I can reproduce it locally.

File: test/resources/selinux.py
Comment (ggiguash) [inline test/resources/selinux.py:1]: These changes do not seem to be related to the main task, or are they?

File: test/scenarios-periodics/el92-src@rpm-standard1.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-liveimg.ks.template ""
+    launch_vm host1 "rhel-9.2-microshift-source"
+    configure_vm_firewall host1
```
Comment (ggiguash) [inline test/scenarios-periodics/el92-src@rpm-standard1.sh:8]: Why do we need this here? What's different from other scenarios?

File: test/scenarios-periodics/el92-src@rpm-standard1.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    run_command_on_vm host1 "sudo systemctl enable --now microshift"
```
Comment (ggiguash) [inline test/scenarios-periodics/el92-src@rpm-standard1.sh:9]: I thought we were enabling microshift service in the source base template. Are we losing this setting somehow?
If so, should we also enable the `microshift-test-agent` service here?

File: test/resources/selinux.py
Comment (dhellmann) [reply test/resources/selinux.py:1]: The commit message explains. https://github.com/openshift/microshift/pull/2850/commits/d1396b335ff61d4c865abb911b069ad289a51900

I get different results from the fcontext list on RPM and ostree systems. The test was overly rigid, testing for an exact match. The important thing to do is ensure that the rules that are specific to MicroShift are installed properly, so I rewrote the test to do that so it would work on both deployment types and be more resilient to changes in the future.

File: test/scenarios-periodics/el92-src@rpm-standard1.sh (hunk 1 above)
Comment (dhellmann) [reply test/scenarios-periodics/el92-src@rpm-standard1.sh:8]: The RPM-based system doesn't have the firewall configured by the blueprint. This is the same function that's used in the RPM installation test scenario.

File: test/scenarios-periodics/el92-src@rpm-standard1.sh (hunk 2 above)
Comment (dhellmann) [reply test/scenarios-periodics/el92-src@rpm-standard1.sh:9]: The blueprint item to enable the service does not apply to the RPM-based system.

The tests do not use the agent service, so it's not being enabled.

File: test/scenarios-periodics/el92-src@rpm-standard1.sh (hunk 1 above)
Comment (ggiguash) [reply test/scenarios-periodics/el92-src@rpm-standard1.sh:8]: We have other scenarios i.e. el92-src@fips.sh using liveimg, but they do not configure firewall explicitly. 
What is different in the current test?

File: test/image-blueprints/layer3-periodic/group3/rhel92-microshift-source.image-installer [hunk 3]
```diff
@@ -0,0 +1 @@
+rhel-9.2-microshift-source
```
[the remark this answers is not in the document]
Comment (dhellmann) [reply test/image-blueprints/layer3-periodic/group3/rhel92-microshift-source.image-installer]: Done
